enhavo/app-bundle/0.9/assets/services/enhavo/enhavo_app.yaml

```diff
@@ -0,0 +1,4 @@
+imports:
+    -
+        path: '@enhavo/app/services/*'
```

Comment:
You dont need this for a bundle. The bundle will register their own services. See https://github.com/enhavo/EnhavoAppBundle/blob/v0.9.2/DependencyInjection/EnhavoAppExtension.php#L46-L61

Reply:
This isn't a configuration file for `Symfony`. It looks similar, but it is used for a webpack plugin within enhavo. (This is how I avoid using Typescript files in the recipe)

https://github.com/enhavo/dependency-injection#import

Reply:
Ah, I see. Okey.